testdata/project-v3-addon/api/v1/admiral_types.go

```diff
@@ -77,7 +77,6 @@ func (o *Admiral) SetCommonStatus(s addonv1alpha1.CommonStatus) {
 }
 
 //+kubebuilder:object:root=true
-//+kubebuilder:resource:scope=Cluster
```

Comment:
Why it was removed?

Reply:
I checked back the plugin template for `*_types.go` and it never had that marker for lists. When we added the cluster scope markers to the addon plugin (#1395) we added it here by mistake.